appinventor/components/src/com/google/appinventor/components/runtime/ButtonBase.java

```diff
@@ -373,8 +374,15 @@ private void updateAppearance() {
         } else {
           // Clear the background image.
           ViewUtil.setBackgroundDrawable(view, null);
-          // Set to the specified color (possibly COLOR_NONE for transparent).
-          TextViewUtil.setBackgroundColor(view, backgroundColor);
+          //Now we set again the default drawable
+          ViewUtil.setBackgroundDrawable(view, defaultButtonDrawable);
+          /* Below removes the nice touch/ripple effect.
```

Comment:
This comment is confusing, as "Below" refers to code that has been removed. Instead change the comment to indicate why you (we) are now using view.getBackground().setColorFilter(). Also check for and remove trailing whitespace. Thanks.

Reply:
I removed the commit now